source/compatibility.tex

```diff
@@ -151,11 +151,11 @@
 \change A name of file scope that is explicitly declared \tcode{const}, and not explicitly
 declared \tcode{extern}, has internal linkage, while in C it would have external linkage
 \rationale
-Because \tcode{const} objects can be used as compile-time values in
+Because \tcode{const} objects may be used as values during translation in
 \Cpp, this feature urges programmers to provide explicit initializer
-values for each \tcode{const}.
-This feature allows the user to put \tcode{const}objects in header files that are included
-in many compilation units.
+for each \tcode{const} object.
+This feature allows the user to put \tcode{const} objects in headers that are included
```

Comment:
That made my day. *Of course* the standard would use "headers" and "source files" in a completely different way from the colloquial use :-)